.github/workflows/test.yml

```diff
@@ -23,17 +19,39 @@ on:
 
 jobs:
   ci:
-    runs-on: <image-name>
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
 
-      - name: Use <setup tooling>
-        uses: <action to setup tooling>
+      - name: Setup Odin
+        uses: laytan/setup-odin@v2
```

Comment:
Exercism prefers having things pinned to specific commits to help reproducibility.

Reply:
Yes, I don't think `laytan/setup-odin` supports commit hashes yet. Let me rather see whether I can pull a specific commit of Odin (or a compatible binary) for this step.

Reply:
Another option is to write a script similar to: https://github.com/exercism/odin/blob/main/bin/fetch-ols-odinfmt.sh

And pull from https://github.com/odin-lang/Odin/releases

Reply:
It does indeed support commit hashes, and I've updated it to the August release.